Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "geyser-grpc-connector"
-version = "0.10.1+yellowstone.1.12"
+version = "2.0.0"
```

Comment:
should have same naming convention

Reply:
Can you eleborate on this naming convention - I don't understand what 0.10.1+yellowstone.1.12 means

Reply:
0.10.1 is the version of the connector. "1.12" is the yellowstone-proto (protobuf) wire format version

Reply:
Okay that sounds like we want 0.11.0+yellowstone.2.0.0 for this

Reply:
ack